tests/test_controller.py

```diff
@@ -62,3 +62,42 @@ def test_resources_route():
     data = json.loads(resp.data)
     assert data[0]["name"] == "Array theory"
     assert data[0]["content"] == "Arrays start counting at zero. Trying to access an empty array may throw an out of bounds error"
+
+
+def test_resources_post_feedback():
```

Comment:
consider extracting and refactoring (possibly thursday?)

Reply:
yaaaaaass!